main.go

```diff
@@ -1,16 +1,21 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+
+	"github.com/hoshina85/test-ci/lib"
+	"github.com/serenize/snaker"
+)
 
 func main() {
 	var s string
 
-	if s, _ := hoge(); s != "" {
+	if s, _ := hello(); s != "" {
```

Comment:
__[gometalinter]__
* [vetshadow] declaration of "s" shadows declaration at main.go:11

<!-- Comment by [SideCI](https://sideci.com) -->